passage1/chain.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "passage",
+  "chain_id": "passage1",
```

Comment:
Sorry if I wasn't clear. chain_name should be passage1, and chain_id should remain as passage-1